tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -1,26 +1,120 @@
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-178
+    ${cpu_usage_before} =    Read Current CPU Usage
+    Run Jupyter Notebook For 5 Minutes
+    ${cpu_usage_after} =    Read Current CPU Usage
+    Should Not Be Equal    ${cpu_usage_before}    ${cpu_usage_after}
+
+Test Metric "Rhods_Total_Users" On ODS Prometheus
+    [Documentation]    Verifies that metrics rhods_total_users and jupyterhub_total_users have the same value in Prometheus
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-628
+    ${expression} =    Set Variable    rhods_total_users&step=1  #step = 1 beacuase it will return value of current second
+    ${rhods_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${expression}
+    ${expression} =    Set Variable    jupyterhub_total_users&step=1  #step = 1 beacuase it will return value of current second
+    ${jupyterhub_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}
+    ...    ${expression}
+    Should Be Equal    ${rhods_total_users.json()["data"]["result"][0]["value"][-1]}
+    ...    ${jupyterhub_total_users.json()["data"]["result"][0]["value"][-1]}
+    ...    msg=metrics rhods_total_users and jupyterhub_total_users don't have the same value
+
+Test Metric Existence For "Rhods_Aggregate_Availability" On ODS Prometheus
+    [Documentation]    Verifies the rhods aggregate availability on rhods prometheus
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-636
+    ${expression} =    Set Variable    rhods_aggregate_availability&step=1
+    ${resp} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${expression}
+    Should Be Equal    ${resp.json()["data"]["result"][0]["value"][-1]}    1
+    ...    msg=metric rhods_aggregate_availability should be 1
+
 
 *** Keywords ***
+Begin Metrics Web Test
+    Set Library Search Order    SeleniumLibrary
+
+End Metrics Web Test
+    Close All Browsers
 
 Check Prometheus Recording Rules
-  Prometheus.Verify Rules  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  record  @{RECORD_GROUPS}
+    Prometheus.Verify Rules    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    record    @{RECORD_GROUPS}
 
 Check Prometheus Alerting Rules
-  Prometheus.Verify Rules  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  alert  @{ALERT_GROUPS}
+    Prometheus.Verify Rules    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    alert    @{ALERT_GROUPS}
+
+Read Current CPU Usage
+    [Documentation]    Returns list of current cpu usage
+    ${version_check} =    Is RHODS Version Greater Or Equal Than    1.5.0
+    IF    ${version_check}==True
+        ${Expression} =    Set Variable
+        ...    sum(rate(container_cpu_usage_seconds_total{prometheus_replica="prometheus-k8s-0", container="",pod=~"jupyterhub-nb.*",namespace="rhods-notebooks"}[1h]))
+    ELSE
```

Comment:
should this check be removed since v1.5.0 is not supported anymore?

Reply:
yes I have removed
Done